### PR TITLE
chore: NextJS upgraded to 13.4

### PR DESCRIPTION
**Pull request by emrysal**
## What does this PR do?

Upgrades NextJS to 13.4

### CODE REVIEW

File: apps/swagger/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "dependencies": {
     "highlight.js": "^11.6.0",
     "isarray": "2.0.5",
-    "next": "^13.2.1",
+    "next": "~13.2.1",
```
Comment (emrysal) [inline apps/swagger/package.json:17]: Lock swagger to 13.2, no reason for 13.4

File: apps/web/package.json [hunk 2]
```diff
@@ -86,7 +86,7 @@
     "memory-cache": "^0.2.0",
     "micro": "^10.0.1",
     "mime-types": "^2.1.35",
-    "next": "^13.2.1",
+    "next": "^13.4.0",
```
Comment (emrysal) [inline apps/web/package.json:89]: Upgrade for clarity, yarn up would be enough.

File: packages/lib/package.json [hunk 3]
```diff
@@ -20,6 +20,7 @@
     "ical.js": "^1.4.0",
     "ics": "^2.37.0",
     "jimp": "^0.16.1",
+    "next-collect": "^0.2.1",
```
Comment (emrysal) [inline packages/lib/package.json:23]: Missing dep

File: packages/ui/package.json [hunk 4]
```diff
@@ -30,7 +30,8 @@
     "@wojtekmaj/react-daterange-picker": "^3.3.1",
     "class-variance-authority": "^0.4.0",
     "downshift": "^6.1.9",
-    "next": "^13.2.1",
+    "next": "^13.4.0",
+    "next-seo": "^4.26.0",
```
Comment (emrysal) [inline packages/ui/package.json:34]: Missing dep

**Review by emrysal — COMMENTED** [review]
Self review done

File: apps/swagger/package.json (hunk 1 above)
Comment (zomars) [reply apps/swagger/package.json:17]: Why not upgrade as well? That way we prevent two next.js versions on the lock file.

Comment (emrysal) [reply apps/swagger/package.json:17]: I want to isolate this for now so I can more thoroughly test separate next versions - so we can stagger things in the future when we grow and have more apps.

**Review by zomars — APPROVED** [review]

**Review by pumfleet — APPROVED** [review]

**Review by keithwillcode — CHANGES_REQUESTED** [review]
Same E2E tests are failing consistently.

**Comment by emrysal** [comment]
Closing this in favour of https://github.com/calcom/cal.com/pull/9578 - but the underlying issue this could not be merged is still the same. Tests aren't passing @keithwillcode.

**Pull request closed without merging**